<!--?xml version="1.0" encoding="UTF-8"?-->
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en">
 <head>
  <title>Quantitative Biology  authors/titles "new"</title>
  <link rel="shortcut icon" href="/favicon.ico" type="image/x-icon">
  <link rel="stylesheet" type="text/css" media="screen" href="//static.arxiv.org/static/browse/0.3.2.8/css/arXiv.css?v=20220215">
  <link rel="stylesheet" type="text/css" media="screen" href="/bibex/bibex.css?20181009">
  <link rel="stylesheet" type="text/css" media="screen" href="https://static.arxiv.org/static/browse/0.3.8/css/browse_search.css">
  <link rel="alternate" type="application/rss+xml" title="Quantitative Biology " href="http://arxiv.org/rss/q-bio">
  <script src="/js/mathjaxToggle.min.js" type="text/javascript"></script>
 </head>
 <body class="with-cu-identity">
  <div id="cu-identity">
   <div id="cu-logo">
    <a href="https://www.cornell.edu/"><img src="//static.arxiv.org/icons/cu/cornell-reduced-white-SMALL.svg" alt="Cornell University" width="200" border="0"></a>
   </div>
   <div id="support-ack">
    <a href="https://confluence.cornell.edu/x/ALlRF">We gratefully acknowledge support from<br>
      the Simons Foundation and member institutions.</a>
   </div>
  </div>
  <div id="header">
   <h1 class="header-breadcrumbs"><a href="/"><img src="//static.arxiv.org/images/arxiv-logo-one-color-white.svg" aria-label="logo" alt="arxiv logo" width="85" style="width:85px;margin-right:8px;"></a> <span>&gt;</span> <a href="/list/q-bio/recent">q-bio</a></h1>
   <div class="search-block level-right">
    <form class="level-item mini-search" method="GET" action="https://arxiv.org/search" _lpchecked="1">
     <div class="field has-addons">
      <div class="control">
       <input class="input is-small" type="text" name="query" placeholder="Search..." aria-label="Search term or terms" data-com.agilebits.onepassword.user-edited="yes">
       <p class="help"><a href="https://arxiv.org/help">Help</a> | <a href="https://arxiv.org/search/advanced">Advanced Search</a></p>
      </div>
      <div class="control">
       <div class="select is-small">
        <select name="searchtype" aria-label="Field to search"> <option value="all" selected>All fields</option> <option value="title">Title</option> <option value="author">Author</option> <option value="abstract">Abstract</option> <option value="comments">Comments</option> <option value="journal_ref">Journal reference</option> <option value="acm_class">ACM classification</option> <option value="msc_class">MSC classification</option> <option value="report_num">Report number</option> <option value="paper_id">arXiv identifier</option> <option value="doi">DOI</option> <option value="orcid">ORCID</option> <option value="author_id">arXiv author ID</option> <option value="help">Help pages</option> <option value="full_text">Full text</option> </select>
       </div>
      </div><button class="button is-small is-cul-darker">Search</button>
     </div>
    </form>
   </div>
  </div>
  <div id="content">
   <div id="dlpage">
    <h1>Quantitative Biology</h1>
    <h2>New submissions</h2>
    <div class="list-dateline">
     Submissions received from Mon 29 Apr 24 to Tue 30 Apr 24, announced Wed, 1 May 24
    </div>
    <ul>
     <li><a href="/list/q-bio/new?skip=0&amp;show=2000">New submissions</a></li>
     <li><a href="#item9">Cross-lists</a></li>
     <li><a href="#item14">Replacements</a></li>
    </ul><small>[ total of 25 entries: <b>1-25</b> ]</small>
    <br><small>[ showing up to 2000 entries per page: <a href="/list/q-bio/new?skip=0&amp;show=1000">fewer</a> | <font color="#999999">more</font> ]</small>
    <br>
    <h3>New submissions for Wed, 1 May 24</h3>
    <dl>
     <dt>
      <a name="item1">[1]</a>&nbsp; <span class="list-identifier"><a href="/abs/2404.18960" title="Abstract">arXiv:2404.18960</a> [<a href="/pdf/2404.18960" title="Download PDF">pdf</a>, <a href="/ps/2404.18960" title="Download PostScript">ps</a>, <a href="/format/2404.18960" title="Other formats">other</a>]</span>
     </dt>
     <dd>
      <div class="meta">
       <div class="list-title mathjax">
        <span class="descriptor">Title:</span> Leak Proof CMap; a framework for training and evaluation of cell line agnostic L1000 similarity methods
       </div>
       <div class="list-authors">
        <span class="descriptor">Authors:</span> <a href="/search/q-bio?searchtype=author&amp;query=Shave%2C+S">Steven Shave</a>, <a href="/search/q-bio?searchtype=author&amp;query=Kasprowicz%2C+R">Richard Kasprowicz</a>, <a href="/search/q-bio?searchtype=author&amp;query=Athar%2C+A+M">Abdullah M. Athar</a>, <a href="/search/q-bio?searchtype=author&amp;query=Vlachou%2C+D">Denise Vlachou</a>, <a href="/search/q-bio?searchtype=author&amp;query=Carragher%2C+N+O">Neil O. Carragher</a>, <a href="/search/q-bio?searchtype=author&amp;query=Nguyen%2C+C+Q">Cuong Q. Nguyen</a>
       </div>
       <div class="list-subjects">
        <span class="descriptor">Subjects:</span> <span class="primary-subject">Quantitative Methods (q-bio.QM)</span>; Machine Learning (cs.LG)
       </div>
       <p class="mathjax">The Connectivity Map (CMap) is a large publicly available database of cellular transcriptomic responses to chemical and genetic perturbations built using a standardized acquisition protocol known as the L1000 technique. Databases such as CMap provide an exciting opportunity to enrich drug discovery efforts, providing a 'known' phenotypic landscape to explore and enabling the development of state of the art techniques for enhanced information extraction and better informed decisions. Whilst multiple methods for measuring phenotypic similarity and interrogating profiles have been developed, the field is severely lacking standardized benchmarks using appropriate data splitting for training and unbiased evaluation of machine learning methods. To address this, we have developed 'Leak Proof CMap' and exemplified its application to a set of common transcriptomic and generic phenotypic similarity methods along with an exemplar triplet loss-based method. Benchmarking in three critical performance areas (compactness, distinctness, and uniqueness) is conducted using carefully crafted data splits ensuring no similar cell lines or treatments with shared or closely matching responses or mechanisms of action are present in training, validation, or test sets. This enables testing of models with unseen samples akin to exploring treatments with novel modes of action in novel patient derived cell lines. With a carefully crafted benchmark and data splitting regime in place, the tooling now exists to create performant phenotypic similarity methods for use in personalized medicine (novel cell lines) and to better augment high throughput phenotypic screening technologies with the L1000 transcriptomic technology.</p>
      </div>
     </dd>
     <dt>
      <a name="item2">[2]</a>&nbsp; <span class="list-identifier"><a href="/abs/2404.18970" title="Abstract">arXiv:2404.18970</a> [<a href="/pdf/2404.18970" title="Download PDF">pdf</a>, <a href="/ps/2404.18970" title="Download PostScript">ps</a>, <a href="/format/2404.18970" title="Other formats">other</a>]</span>
     </dt>
     <dd>
      <div class="meta">
       <div class="list-title mathjax">
        <span class="descriptor">Title:</span> A pretest-posttest pilot study for augmented reality-based physical-cognitive training in community-dwelling older adults at risk of mild cognitive impairment
       </div>
       <div class="list-authors">
        <span class="descriptor">Authors:</span> <a href="/search/q-bio?searchtype=author&amp;query=Chaipunko%2C+S">Sirinun Chaipunko</a>, <a href="/search/q-bio?searchtype=author&amp;query=Ammawat%2C+W">Watthanaree Ammawat</a>, <a href="/search/q-bio?searchtype=author&amp;query=Oanmun%2C+K">Keerathi Oanmun</a>, <a href="/search/q-bio?searchtype=author&amp;query=Hongnaphadol%2C+W">Wanvipha Hongnaphadol</a>, <a href="/search/q-bio?searchtype=author&amp;query=Sorasak%2C+S">Supatida Sorasak</a>, <a href="/search/q-bio?searchtype=author&amp;query=Makmee%2C+P">Pattrawadee Makmee</a>
       </div>
       <div class="list-comments mathjax">
        <span class="descriptor">Comments:</span> 20 pages, 3 figures
       </div>
       <div class="list-subjects">
        <span class="descriptor">Subjects:</span> <span class="primary-subject">Neurons and Cognition (q-bio.NC)</span>
       </div>
       <p class="mathjax">As cognitive interventions for older adults evolve, modern technologies are increasingly integrated into their development. This study investigates the efficacy of augmented reality (AR)-based physical-cognitive training using an interactive game with Kinect motion sensor technology on older individuals at risk of mild cognitive impairment. Utilizing a pretest-posttest experimental design, twenty participants (mean age 66.8 SD. = 4.6 years, age range 60-78 years) underwent eighteen individual training sessions, lasting 45 to 60 minutes each, conducted three times a week over a span of 1.5 months. The training modules from five activities, encompassing episodic and working memory, attention and inhibition, cognitive flexibility, and speed processing, were integrated with physical movement and culturally relevant Thai-context activities. Results revealed significant improvements in inhibition, cognitive flexibility, accuracy, and reaction time, with working memory demonstrating enhancements in accuracy albeit not in reaction time. These findings underscore the potential of AR interventions to bolster basic executive enhancement among community-dwelling older adults at risk of cognitive decline.</p>
      </div>
     </dd>
     <dt>
      <a name="item3">[3]</a>&nbsp; <span class="list-identifier"><a href="/abs/2404.18973" title="Abstract">arXiv:2404.18973</a> [<a href="/pdf/2404.18973" title="Download PDF">pdf</a>, <a href="/ps/2404.18973" title="Download PostScript">ps</a>, <a href="/format/2404.18973" title="Other formats">other</a>]</span>
     </dt>
     <dd>
      <div class="meta">
       <div class="list-title mathjax">
        <span class="descriptor">Title:</span> The Convergence of AI and Synthetic Biology: The Looming Deluge
       </div>
       <div class="list-authors">
        <span class="descriptor">Authors:</span> <a href="/search/q-bio?searchtype=author&amp;query=Vindman%2C+C">Cindy Vindman</a>, <a href="/search/q-bio?searchtype=author&amp;query=Trump%2C+B">Benjamin Trump</a>, <a href="/search/q-bio?searchtype=author&amp;query=Cummings%2C+C">Christopher Cummings</a>, <a href="/search/q-bio?searchtype=author&amp;query=Smith%2C+M">Madison Smith</a>, <a href="/search/q-bio?searchtype=author&amp;query=Titus%2C+A+J">Alexander J. Titus</a>, <a href="/search/q-bio?searchtype=author&amp;query=Oye%2C+K">Ken Oye</a>, <a href="/search/q-bio?searchtype=author&amp;query=Prado%2C+V">Valentina Prado</a>, <a href="/search/q-bio?searchtype=author&amp;query=Turmus%2C+E">Eyup Turmus</a>, <a href="/search/q-bio?searchtype=author&amp;query=Linkov%2C+I">Igor Linkov</a>
       </div>
       <div class="list-subjects">
        <span class="descriptor">Subjects:</span> <span class="primary-subject">Other Quantitative Biology (q-bio.OT)</span>
       </div>
       <p class="mathjax">The convergence of artificial intelligence (AI) and synthetic biology is rapidly accelerating the pace of biological discovery and engineering. AI techniques, such as large language models and biological design tools, are enabling the automated design, build, test, and learning cycles for engineered biological systems. This convergence promises to democratize synthetic biology and unlock novel applications across domains from medicine to environmental sustainability. However, it also poses significant risks around reliability, dual use, and governance. The opacity of AI models, the deskilling of workforces, and the outdated nature of current regulatory frameworks present challenges in ensuring responsible development. Urgent attention is needed to update governance structures, integrate human oversight into increasingly automated workflows, and foster a culture of responsibility among the growing community of bioengineers. Only by proactively addressing these issues can we realize the transformative potential of AI-driven synthetic biology while mitigating its risks.</p>
      </div>
     </dd>
     <dt>
      <a name="item4">[4]</a>&nbsp; <span class="list-identifier"><a href="/abs/2404.19041" title="Abstract">arXiv:2404.19041</a> [<a href="/pdf/2404.19041" title="Download PDF">pdf</a>, <a href="/format/2404.19041" title="Other formats">other</a>]</span>
     </dt>
     <dd>
      <div class="meta">
       <div class="list-title mathjax">
        <span class="descriptor">Title:</span> Stochastic dynamics of two-compartment models with regulatory mechanisms for hematopoiesis
       </div>
       <div class="list-authors">
        <span class="descriptor">Authors:</span> <a href="/search/q-bio?searchtype=author&amp;query=Wang%2C+R">Ren-Yi Wang</a>, <a href="/search/q-bio?searchtype=author&amp;query=Kimmel%2C+M">Marek Kimmel</a>, <a href="/search/q-bio?searchtype=author&amp;query=Pang%2C+G">Guodong Pang</a>
       </div>
       <div class="list-subjects">
        <span class="descriptor">Subjects:</span> <span class="primary-subject">Populations and Evolution (q-bio.PE)</span>
       </div>
       <p class="mathjax">We present an asymptotic analysis of a stochastic two-compartmental cell proliferation system with regulatory mechanisms. We model the system as a state-dependent birth and death process. Proliferation of hematopoietic stem cells (HSCs) is regulated by population density of HSC-derived clones and differentiation of HSC is regulated by population density of HSCs. By scaling up the initial population, we show the density of dynamics converges in distribution to the solution of a system of ordinary differential equations. The system of ODE has a unique non-trivial equilibrium that is globally stable. Furthermore, we show the scaled fluctuation of the population converges in law to a linear diffusion. With initial data being Gaussian, the limit is a Gauss-Markov process and we prove the process will stabilize exponentially fast in the 2-Wasserstein metric. We apply our results to analyze and compare two regulatory mechanisms in the hematopoietic system. Simulations are conducted to verify our large-scale and long-time approximation of the dynamics. We demonstrate some regulatory mechanisms are efficient (converge to steady state rapidly) but not effective (have large fluctuation around the steady state).</p>
      </div>
     </dd>
     <dt>
      <a name="item5">[5]</a>&nbsp; <span class="list-identifier"><a href="/abs/2404.19230" title="Abstract">arXiv:2404.19230</a> [<a href="/pdf/2404.19230" title="Download PDF">pdf</a>, <a href="/ps/2404.19230" title="Download PostScript">ps</a>, <a href="/format/2404.19230" title="Other formats">other</a>]</span>
     </dt>
     <dd>
      <div class="meta">
       <div class="list-title mathjax">
        <span class="descriptor">Title:</span> Deep Lead Optimization: Leveraging Generative AI for Structural Modification
       </div>
       <div class="list-authors">
        <span class="descriptor">Authors:</span> <a href="/search/q-bio?searchtype=author&amp;query=Zhang%2C+O">Odin Zhang</a>, <a href="/search/q-bio?searchtype=author&amp;query=Lin%2C+H">Haitao Lin</a>, <a href="/search/q-bio?searchtype=author&amp;query=Zhang%2C+H">Hui Zhang</a>, <a href="/search/q-bio?searchtype=author&amp;query=Zhao%2C+H">Huifeng Zhao</a>, <a href="/search/q-bio?searchtype=author&amp;query=Huang%2C+Y">Yufei Huang</a>, <a href="/search/q-bio?searchtype=author&amp;query=Huang%2C+Y">Yuansheng Huang</a>, <a href="/search/q-bio?searchtype=author&amp;query=Jiang%2C+D">Dejun Jiang</a>, <a href="/search/q-bio?searchtype=author&amp;query=Hsieh%2C+C">Chang-yu Hsieh</a>, <a href="/search/q-bio?searchtype=author&amp;query=Pan%2C+P">Peichen Pan</a>, <a href="/search/q-bio?searchtype=author&amp;query=Hou%2C+T">Tingjun Hou</a>
       </div>
       <div class="list-subjects">
        <span class="descriptor">Subjects:</span> <span class="primary-subject">Biomolecules (q-bio.BM)</span>; Artificial Intelligence (cs.AI)
       </div>
       <p class="mathjax">The idea of using deep-learning-based molecular generation to accelerate discovery of drug candidates has attracted extraordinary attention, and many deep generative models have been developed for automated drug design, termed molecular generation. In general, molecular generation encompasses two main strategies: de novo design, which generates novel molecular structures from scratch, and lead optimization, which refines existing molecules into drug candidates. Among them, lead optimization plays an important role in real-world drug design. For example, it can enable the development of me-better drugs that are chemically distinct yet more effective than the original drugs. It can also facilitate fragment-based drug design, transforming virtual-screened small ligands with low affinity into first-in-class medicines. Despite its importance, automated lead optimization remains underexplored compared to the well-established de novo generative models, due to its reliance on complex biological and chemical knowledge. To bridge this gap, we conduct a systematic review of traditional computational methods for lead optimization, organizing these strategies into four principal sub-tasks with defined inputs and outputs. This review delves into the basic concepts, goals, conventional CADD techniques, and recent advancements in AIDD. Additionally, we introduce a unified perspective based on constrained subgraph generation to harmonize the methodologies of de novo design and lead optimization. Through this lens, de novo design can incorporate strategies from lead optimization to address the challenge of generating hard-to-synthesize molecules; inversely, lead optimization can benefit from the innovations in de novo design by approaching it as a task of generating molecules conditioned on certain substructures.</p>
      </div>
     </dd>
     <dt>
      <a name="item6">[6]</a>&nbsp; <span class="list-identifier"><a href="/abs/2404.19235" title="Abstract">arXiv:2404.19235</a> [<a href="/pdf/2404.19235" title="Download PDF">pdf</a>, <a href="/format/2404.19235" title="Other formats">other</a>]</span>
     </dt>
     <dd>
      <div class="meta">
       <div class="list-title mathjax">
        <span class="descriptor">Title:</span> Computational Approaches of Modelling Human Papillomavirus Transmission and Prevention Strategies: A Systematic Review
       </div>
       <div class="list-authors">
        <span class="descriptor">Authors:</span> <a href="/search/q-bio?searchtype=author&amp;query=Wang%2C+W">Weiyi Wang</a>, <a href="/search/q-bio?searchtype=author&amp;query=Sawleshwarkar%2C+S">Shailendra Sawleshwarkar</a>, <a href="/search/q-bio?searchtype=author&amp;query=Piraveenan%2C+M">Mahendra Piraveenan</a>
       </div>
       <div class="list-subjects">
        <span class="descriptor">Subjects:</span> <span class="primary-subject">Populations and Evolution (q-bio.PE)</span>
       </div>
       <p class="mathjax">Human papillomavirus (HPV) infection is the most common sexually transmitted infection in the world. Persistent oncogenic Human papillomavirus infection has been a leading threat to global health and can lead to serious complications such as cervical cancer. Prevention interventions including vaccination and screening have been proved effective in reducing the risk of HPV-related diseases. In recent decades, computational epidemiology has been serving as a very useful tool to study HPV transmission dynamics and evaluation of prevention strategies. In this paper, we conduct a comprehensive literature review on state-of-the-art computational epidemic models for HPV disease dynamics, transmission dynamics, as well as prevention efforts. We summarise current research trends, identify gaps in the present literature, and identify future research directions with potential in accelerating the containment and/or elimination of HPV infection.</p>
      </div>
     </dd>
     <dt>
      <a name="item7">[7]</a>&nbsp; <span class="list-identifier"><a href="/abs/2404.19309" title="Abstract">arXiv:2404.19309</a> [<a href="/pdf/2404.19309" title="Download PDF">pdf</a>, <a href="/ps/2404.19309" title="Download PostScript">ps</a>, <a href="/format/2404.19309" title="Other formats">other</a>]</span>
     </dt>
     <dd>
      <div class="meta">
       <div class="list-title mathjax">
        <span class="descriptor">Title:</span> Noise propagation and MP-PCA image denoising for high-resolution quantitative T2* and magnetic susceptibility mapping (QSM)
       </div>
       <div class="list-authors">
        <span class="descriptor">Authors:</span> <a href="/search/q-bio?searchtype=author&amp;query=Doniza%2C+L">Liad Doniza</a> (1), <a href="/search/q-bio?searchtype=author&amp;query=Lee%2C+M">Mitchel Lee</a> (2), <a href="/search/q-bio?searchtype=author&amp;query=Katzir%2C+T+B">Tamar Blumenfeld Katzir</a> (3), <a href="/search/q-bio?searchtype=author&amp;query=Artzi%2C+M">Moran Artzi</a> (4,5,6), <a href="/search/q-bio?searchtype=author&amp;query=Bashat%2C+D+B">Dafna Ben Bashat</a> (4,5,6), <a href="/search/q-bio?searchtype=author&amp;query=Radunsky%2C+D">Dvir Radunsky</a> (3), <a href="/search/q-bio?searchtype=author&amp;query=Shmueli%2C+K">Karin Shmueli</a> (2), <a href="/search/q-bio?searchtype=author&amp;query=Ben-Eliezer%2C+N">Noam Ben-Eliezer</a> (3,5,7) ((1) Department of Electrical Engineering, Tel Aviv University, Tel Aviv, Israel, (2) Department of Medical Physics and Biomedical Engineering, University College London, London, UK, (3) Department of Biomedical Engineering, Tel Aviv University, Tel Aviv, Israel, (4) Sagol Brain Institute, Tel Aviv Medical Center, Tel Aviv, Israel, (5) Sagol School of Neuroscience, Tel Aviv University, Tel-Aviv, Israel, (6) Sackler Faculty of Medicine, Tel Aviv University, Tel Aviv, Israel, (7) Center for Advanced Imaging Innovation and Research (CAI2R), New-York University Langone Medical Center, New York, NY, United States)
       </div>
       <div class="list-comments mathjax">
        <span class="descriptor">Comments:</span> 9 pages, 8 figures, 3 tables. It was accepted to be presented in a peer-reviewed annual ISMRM meeting, which will be held in Singapore in May 2024
       </div>
       <div class="list-subjects">
        <span class="descriptor">Subjects:</span> <span class="primary-subject">Quantitative Methods (q-bio.QM)</span>
       </div>
       <p class="mathjax">Quantitative Susceptibility Mapping (QSM) is a technique for measuring magnetic susceptibility of tissues, aiding in the detection of pathologies like traumatic brain injury and multiple sclerosis by analyzing variations in substances such as iron and calcium. Despite its clinical value, achieving high-resolution QSM (voxel sizes &lt; 1 mm3) reduces signal-to-noise ratio (SNR), compromising diagnostic quality. To mitigate this, we applied the Marchenko-Pastur Principal Component Analysis (MP-PCA) denoising technique on T2* weighted data, to enhance the quality of R2*, T2*, and QSM maps. Denoising was tested on a numerical phantom, healthy subjects, and patients with brain metastases and sickle cell disease, demonstrating effective and robust improvements across different scan settings. Further analysis examined noise propagation in R2* and T2* values, revealing lower noise-related variations in R2* values compared to T2* values which tended to be overestimated due to noise. Reduced variability was observed in QSM values post denoising, demonstrating MP-PCA's potential to improve the</p>
      </div>
     </dd>
     <dt>
      <a name="item8">[8]</a>&nbsp; <span class="list-identifier"><a href="/abs/2404.19739" title="Abstract">arXiv:2404.19739</a> [<a href="/pdf/2404.19739" title="Download PDF">pdf</a>, <a href="/format/2404.19739" title="Other formats">other</a>]</span>
     </dt>
     <dd>
      <div class="meta">
       <div class="list-title mathjax">
        <span class="descriptor">Title:</span> Mixed Continuous and Categorical Flow Matching for 3D De Novo Molecule Generation
       </div>
       <div class="list-authors">
        <span class="descriptor">Authors:</span> <a href="/search/q-bio?searchtype=author&amp;query=Dunn%2C+I">Ian Dunn</a>, <a href="/search/q-bio?searchtype=author&amp;query=Koes%2C+D+R">David Ryan Koes</a>
       </div>
       <div class="list-subjects">
        <span class="descriptor">Subjects:</span> <span class="primary-subject">Biomolecules (q-bio.BM)</span>; Machine Learning (cs.LG)
       </div>
       <p class="mathjax">Deep generative models that produce novel molecular structures have the potential to facilitate chemical discovery. Diffusion models currently achieve state of the art performance for 3D molecule generation. In this work, we explore the use of flow matching, a recently proposed generative modeling framework that generalizes diffusion models, for the task of de novo molecule generation. Flow matching provides flexibility in model design; however, the framework is predicated on the assumption of continuously-valued data. 3D de novo molecule generation requires jointly sampling continuous and categorical variables such as atom position and atom type. We extend the flow matching framework to categorical data by constructing flows that are constrained to exist on a continuous representation of categorical data known as the probability simplex. We call this extension SimplexFlow. We explore the use of SimplexFlow for de novo molecule generation. However, we find that, in practice, a simpler approach that makes no accommodations for the categorical nature of the data yields equivalent or superior performance. As a result of these experiments, we present FlowMol, a flow matching model for 3D de novo generative model that achieves improved performance over prior flow matching methods, and we raise important questions about the design of prior distributions for achieving strong performance in flow matching models. Code and trained models for reproducing this work are available at https://github.com/dunni3/FlowMol</p>
      </div>
     </dd>
    </dl>
    <h3>Cross-lists for Wed, 1 May 24</h3>
    <dl>
     <dt>
      <a name="item9">[9]</a>&nbsp; <span class="list-identifier"><a href="/abs/2404.19158" title="Abstract">arXiv:2404.19158</a> (cross-list from physics.bio-ph) [<a href="/pdf/2404.19158" title="Download PDF">pdf</a>, <a href="/format/2404.19158" title="Other formats">other</a>]</span>
     </dt>
     <dd>
      <div class="meta">
       <div class="list-title mathjax">
        <span class="descriptor">Title:</span> Polymer Collapse and Liquid-Liquid Phase-Separation are Coupled in a Generalized Prewetting Transition
       </div>
       <div class="list-authors">
        <span class="descriptor">Authors:</span> <a href="/search/physics?searchtype=author&amp;query=Rouches%2C+M+N">Mason N. Rouches</a>, <a href="/search/physics?searchtype=author&amp;query=Machta%2C+B+B">Benjamin B. Machta</a>
       </div>
       <div class="list-comments mathjax">
        <span class="descriptor">Comments:</span> 12 pages, 5 figures
       </div>
       <div class="list-subjects">
        <span class="descriptor">Subjects:</span> <span class="primary-subject">Biological Physics (physics.bio-ph)</span>; Statistical Mechanics (cond-mat.stat-mech); Subcellular Processes (q-bio.SC)
       </div>
       <p class="mathjax">The three-dimensional organization of chromatin is thought to play an important role in controlling gene expression. Specificity in expression is achieved through the interaction of transcription factors and other nuclear proteins with particular sequences of DNA. At unphysiological concentrations many of these nuclear proteins can phase-separate in the absence of DNA, and it has been hypothesized that, in vivo, the thermodynamic forces driving these phases help determine chromosomal organization. However it is unclear how DNA, itself a long polymer subject to configurational transitions, interacts with three-dimensional protein phases. Here we show that a long compressible polymer can be coupled to interacting protein mixtures, leading to a generalized prewetting transition where polymer collapse is coincident with a locally stabilized liquid droplet. We use lattice Monte-Carlo simulations and a mean-field theory to show that these phases can be stable even in regimes where both polymer collapse and coexisting liquid phases are unstable in isolation, and that these new transitions can be either abrupt or continuous. For polymers with internal linear structure we further show that changes in the concentration of bulk components can lead to changes in three-dimensional polymer structure. In the nucleus there are many distinct proteins that interact with many different regions of chromatin, potentially giving rise to many different Prewet phases. The simple systems we consider here highlight chromatin's role as a lower-dimensional surface whose interactions with proteins are required for these novel phases.</p>
      </div>
     </dd>
     <dt>
      <a name="item10">[10]</a>&nbsp; <span class="list-identifier"><a href="/abs/2404.19313" title="Abstract">arXiv:2404.19313</a> (cross-list from quant-ph) [<a href="/pdf/2404.19313" title="Download PDF">pdf</a>, <a href="/format/2404.19313" title="Other formats">other</a>]</span>
     </dt>
     <dd>
      <div class="meta">
       <div class="list-title mathjax">
        <span class="descriptor">Title:</span> High-precision chemical quantum sensing in flowing monodisperse microdroplets
       </div>
       <div class="list-authors">
        <span class="descriptor">Authors:</span> <a href="/search/quant-ph?searchtype=author&amp;query=Sarkar%2C+A">Adrisha Sarkar</a>, <a href="/search/quant-ph?searchtype=author&amp;query=Jones%2C+Z">Zachary Jones</a>, <a href="/search/quant-ph?searchtype=author&amp;query=Parashar%2C+M">Madhur Parashar</a>, <a href="/search/quant-ph?searchtype=author&amp;query=Druga%2C+E">Emanuel Druga</a>, <a href="/search/quant-ph?searchtype=author&amp;query=Akkiraju%2C+A">Amala Akkiraju</a>, <a href="/search/quant-ph?searchtype=author&amp;query=Conti%2C+S">Sophie Conti</a>, <a href="/search/quant-ph?searchtype=author&amp;query=Krishnamoorthi%2C+P">Pranav Krishnamoorthi</a>, <a href="/search/quant-ph?searchtype=author&amp;query=Nachuri%2C+S">Srisai Nachuri</a>, <a href="/search/quant-ph?searchtype=author&amp;query=Aman%2C+P">Parker Aman</a>, <a href="/search/quant-ph?searchtype=author&amp;query=Hashemi%2C+M">Mohammad Hashemi</a>, <a href="/search/quant-ph?searchtype=author&amp;query=Nunn%2C+N">Nicholas Nunn</a>, <a href="/search/quant-ph?searchtype=author&amp;query=Torelli%2C+M">Marco Torelli</a>, <a href="/search/quant-ph?searchtype=author&amp;query=Gilbert%2C+B">Benjamin Gilbert</a>, <a href="/search/quant-ph?searchtype=author&amp;query=Wilson%2C+K+R">Kevin R. Wilson</a>, <a href="/search/quant-ph?searchtype=author&amp;query=Shenderova%2C+O">Olga Shenderova</a>, <a href="/search/quant-ph?searchtype=author&amp;query=Tanjore%2C+D">Deepti Tanjore</a>, <a href="/search/quant-ph?searchtype=author&amp;query=Ajoy%2C+A">Ashok Ajoy</a>
       </div>
       <div class="list-subjects">
        <span class="descriptor">Subjects:</span> <span class="primary-subject">Quantum Physics (quant-ph)</span>; Mesoscale and Nanoscale Physics (cond-mat.mes-hall); Applied Physics (physics.app-ph); Quantitative Methods (q-bio.QM)
       </div>
       <p class="mathjax">We report on a novel flow-based method for high-precision chemical detection that integrates quantum sensing with droplet microfluidics. We deploy nanodiamond particles hosting fluorescent nitrogen vacancy defects as quantum sensors in flowing, monodisperse, picoliter-volume microdroplets containing analyte molecules. ND motion within these microcompartments facilitates close sensor-analyte interaction and mitigates particle heterogeneity. Microdroplet flow rates are rapid (upto 4cm/s) and with minimal drift. Pairing this controlled flow with microwave control of NV electronic spins, we introduce a new noise-suppressed mode of Optically Detected Magnetic Resonance that is sensitive to chemical analytes while resilient against experimental variations, achieving detection of analyte-induced signals at an unprecedented level of a few hundredths of a percent of the ND fluorescence. We demonstrate its application to detecting paramagnetic ions in droplets with simultaneously low limit-of-detection and low analyte volumes, in a manner significantly better than existing technologies. This is combined with exceptional measurement stability over &gt;103s and across hundreds of thousands of droplets, while utilizing minimal sensor volumes and incurring low ND costs (&lt;$0.70 for an hour of operation). Additionally, we demonstrate using these droplets as micro-confinement chambers by co-encapsulating ND quantum sensors with analytes, including single cells. This versatility suggests wide-ranging applications, like single-cell metabolomics and real-time intracellular measurements in bioreactors. Our work paves the way for portable, high-sensitivity, amplification-free, chemical assays with high throughput; introduces a new chemical imaging tool for probing chemical reactions in microenvironments; and establishes the foundation for developing movable, arrayed quantum sensors through droplet microfluidics.</p>
      </div>
     </dd>
     <dt>
      <a name="item11">[11]</a>&nbsp; <span class="list-identifier"><a href="/abs/2404.19328" title="Abstract">arXiv:2404.19328</a> (cross-list from cs.CL) [<a href="/pdf/2404.19328" title="Download PDF">pdf</a>, <a href="/format/2404.19328" title="Other formats">other</a>]</span>
     </dt>
     <dd>
      <div class="meta">
       <div class="list-title mathjax">
        <span class="descriptor">Title:</span> Computational Approaches for Integrating out Subjectivity in Cognate Synonym Selection
       </div>
       <div class="list-authors">
        <span class="descriptor">Authors:</span> <a href="/search/cs?searchtype=author&amp;query=H%C3%A4user%2C+L">Luise Häuser</a>, <a href="/search/cs?searchtype=author&amp;query=J%C3%A4ger%2C+G">Gerhard Jäger</a>, <a href="/search/cs?searchtype=author&amp;query=Stamatakis%2C+A">Alexandros Stamatakis</a>
       </div>
       <div class="list-comments mathjax">
        <span class="descriptor">Comments:</span> Experiments available on GitHub (<a href="https://github.com/luisevonderwiese/synonyms">this https URL</a>, <a href="https://github.com/luisevonderwiese/lingdata">this https URL</a>)
       </div>
       <div class="list-subjects">
        <span class="descriptor">Subjects:</span> <span class="primary-subject">Computation and Language (cs.CL)</span>; Populations and Evolution (q-bio.PE)
       </div>
       <p class="mathjax">Working with cognate data involves handling synonyms, that is, multiple words that describe the same concept in a language. In the early days of language phylogenetics it was recommended to select one synonym only. However, as we show here, binary character matrices, which are used as input for computational methods, do allow for representing the entire dataset including all synonyms. Here we address the question how one can and if one should include all synonyms or whether it is preferable to select synonyms a priori. To this end, we perform maximum likelihood tree inferences with the widely used RAxML-NG tool and show that it yields plausible trees when all synonyms are used as input. Furthermore, we show that a priori synonym selection can yield topologically substantially different trees and we therefore advise against doing so. To represent cognate data including all synonyms, we introduce two types of character matrices beyond the standard binary ones: probabilistic binary and probabilistic multi-valued character matrices. We further show that it is dataset-dependent for which character matrix type the inferred RAxML-NG tree is topologically closest to the gold standard. We also make available a Python interface for generating all of the above character matrix types for cognate data provided in CLDF format.</p>
      </div>
     </dd>
     <dt>
      <a name="item12">[12]</a>&nbsp; <span class="list-identifier"><a href="/abs/2404.19467" title="Abstract">arXiv:2404.19467</a> (cross-list from cs.LG) [<a href="/pdf/2404.19467" title="Download PDF">pdf</a>, <a href="/ps/2404.19467" title="Download PostScript">ps</a>, <a href="/format/2404.19467" title="Other formats">other</a>]</span>
     </dt>
     <dd>
      <div class="meta">
       <div class="list-title mathjax">
        <span class="descriptor">Title:</span> Bayesian Functional Connectivity and Graph Convolutional Network for Working Memory Load Classification
       </div>
       <div class="list-authors">
        <span class="descriptor">Authors:</span> <a href="/search/cs?searchtype=author&amp;query=Gangapuram%2C+H">Harshini Gangapuram</a>, <a href="/search/cs?searchtype=author&amp;query=Manian%2C+V">Vidya Manian</a>
       </div>
       <div class="list-subjects">
        <span class="descriptor">Subjects:</span> <span class="primary-subject">Machine Learning (cs.LG)</span>; Signal Processing (eess.SP); Neurons and Cognition (q-bio.NC)
       </div>
       <p class="mathjax">Brain responses related to working memory originate from distinct brain areas and oscillate at different frequencies. EEG signals with high temporal correlation can effectively capture these responses. Therefore, estimating the functional connectivity of EEG for working memory protocols in different frequency bands plays a significant role in analyzing the brain dynamics with increasing memory and cognitive loads, which remains largely unexplored. The present study introduces a Bayesian structure learning algorithm to learn the functional connectivity of EEG in sensor space. Next, the functional connectivity graphs are taken as input to the graph convolutional network to classify the working memory loads. The intrasubject (subject-specific) classification performed on 154 subjects for six different verbal working memory loads produced the highest classification accuracy of 96% and average classification accuracy of 89%, outperforming state-of-the-art classification models proposed in the literature. Furthermore, the proposed Bayesian structure learning algorithm is compared with state-of-the-art functional connectivity estimation methods through intersubject and intrasubject statistical analysis of variance. The results also show that the alpha and theta bands have better classification accuracy than the beta band.</p>
      </div>
     </dd>
     <dt>
      <a name="item13">[13]</a>&nbsp; <span class="list-identifier"><a href="/abs/2404.19710" title="Abstract">arXiv:2404.19710</a> (cross-list from cs.LG) [<a href="/pdf/2404.19710" title="Download PDF">pdf</a>, <a href="/format/2404.19710" title="Other formats">other</a>]</span>
     </dt>
     <dd>
      <div class="meta">
       <div class="list-title mathjax">
        <span class="descriptor">Title:</span> A rank decomposition for the topological classification of neural representations
       </div>
       <div class="list-authors">
        <span class="descriptor">Authors:</span> <a href="/search/cs?searchtype=author&amp;query=Beshkov%2C+K">Kosio Beshkov</a>, <a href="/search/cs?searchtype=author&amp;query=Einevoll%2C+G+T">Gaute T. Einevoll</a>
       </div>
       <div class="list-subjects">
        <span class="descriptor">Subjects:</span> <span class="primary-subject">Machine Learning (cs.LG)</span>; Algebraic Topology (math.AT); Neurons and Cognition (q-bio.NC)
       </div>
       <p class="mathjax">Neural networks can be thought of as applying a transformation to an input dataset. The way in which they change the topology of such a dataset often holds practical significance for many tasks, particularly those demanding non-homeomorphic mappings for optimal solutions, such as classification problems. In this work, we leverage the fact that neural networks are equivalent to continuous piecewise-affine maps, whose rank can be used to pinpoint regions in the input space that undergo non-homeomorphic transformations, leading to alterations in the topological structure of the input dataset. Our approach enables us to make use of the relative homology sequence, with which one can study the homology groups of the quotient of a manifold $\mathcal{M}$ and a subset $A$, assuming some minimal properties on these spaces. <br>
        As a proof of principle, we empirically investigate the presence of low-rank (topology-changing) affine maps as a function of network width and mean weight. We show that in randomly initialized narrow networks, there will be regions in which the (co)homology groups of a data manifold can change. As the width increases, the homology groups of the input manifold become more likely to be preserved. We end this part of our work by constructing highly non-random wide networks that do not have this property and relating this non-random regime to Dale's principle, which is a defining characteristic of biological neural networks. <br>
        Finally, we study simple feedforward networks trained on MNIST, as well as on toy classification and regression tasks, and show that networks manipulate the topology of data differently depending on the continuity of the task they are trained on.</p>
      </div>
     </dd>
    </dl>
    <h3>Replacements for Wed, 1 May 24</h3>
    <dl>
     <dt>
      <a name="item14">[14]</a>&nbsp; <span class="list-identifier"><a href="/abs/2112.05098" title="Abstract">arXiv:2112.05098</a> (replaced) [<a href="/pdf/2112.05098" title="Download PDF">pdf</a>, <a href="/format/2112.05098" title="Other formats">other</a>]</span>
     </dt>
     <dd>
      <div class="meta">
       <div class="list-title mathjax">
        <span class="descriptor">Title:</span> Intraspecific predator interference promotes biodiversity in ecosystems
       </div>
       <div class="list-authors">
        <span class="descriptor">Authors:</span> <a href="/search/q-bio?searchtype=author&amp;query=Kang%2C+J">Ju Kang</a>, <a href="/search/q-bio?searchtype=author&amp;query=Zhang%2C+S">Shijie Zhang</a>, <a href="/search/q-bio?searchtype=author&amp;query=Niu%2C+Y">Yiyuan Niu</a>, <a href="/search/q-bio?searchtype=author&amp;query=Zhong%2C+F">Fan Zhong</a>, <a href="/search/q-bio?searchtype=author&amp;query=Wang%2C+X">Xin Wang</a>
       </div>
       <div class="list-comments mathjax">
        <span class="descriptor">Comments:</span> Main text 14 pages, 3 figures. Appendices 34 pages, 15 Appendix-figures
       </div>
       <div class="list-subjects">
        <span class="descriptor">Subjects:</span> <span class="primary-subject">Populations and Evolution (q-bio.PE)</span>; Statistical Mechanics (cond-mat.stat-mech); Biological Physics (physics.bio-ph)
       </div>
      </div>
     </dd>
     <dt>
      <a name="item15">[15]</a>&nbsp; <span class="list-identifier"><a href="/abs/2304.02587" title="Abstract">arXiv:2304.02587</a> (replaced) [<a href="/pdf/2304.02587" title="Download PDF">pdf</a>, <a href="/format/2304.02587" title="Other formats">other</a>]</span>
     </dt>
     <dd>
      <div class="meta">
       <div class="list-title mathjax">
        <span class="descriptor">Title:</span> A Two-Phase Model of Early Atherosclerotic Plaque Development with LDL Toxicity Effects
       </div>
       <div class="list-authors">
        <span class="descriptor">Authors:</span> <a href="/search/q-bio?searchtype=author&amp;query=Pramanik%2C+A+S">Abdush Salam Pramanik</a>, <a href="/search/q-bio?searchtype=author&amp;query=Dey%2C+B">Bibaswan Dey</a>, <a href="/search/q-bio?searchtype=author&amp;query=Sekhar%2C+G+P+R">G. P. Raja Sekhar</a>
       </div>
       <div class="list-subjects">
        <span class="descriptor">Subjects:</span> <span class="primary-subject">Tissues and Organs (q-bio.TO)</span>; Fluid Dynamics (physics.flu-dyn)
       </div>
      </div>
     </dd>
     <dt>
      <a name="item16">[16]</a>&nbsp; <span class="list-identifier"><a href="/abs/2308.12586" title="Abstract">arXiv:2308.12586</a> (replaced) [<a href="/pdf/2308.12586" title="Download PDF">pdf</a>, <a href="/format/2308.12586" title="Other formats">other</a>]</span>
     </dt>
     <dd>
      <div class="meta">
       <div class="list-title mathjax">
        <span class="descriptor">Title:</span> Network transformation-based analysis of biochemical systems
       </div>
       <div class="list-authors">
        <span class="descriptor">Authors:</span> <a href="/search/q-bio?searchtype=author&amp;query=Talabis%2C+D+A">Dylan Antonio Talabis</a>, <a href="/search/q-bio?searchtype=author&amp;query=Mendoza%2C+E">Eduardo Mendoza</a>
       </div>
       <div class="list-comments mathjax">
        <span class="descriptor">Comments:</span> 40 pages. arXiv admin note: text overlap with <a href="/abs/1109.2923">arXiv:1109.2923</a> by other authors
       </div>
       <div class="list-subjects">
        <span class="descriptor">Subjects:</span> <span class="primary-subject">Molecular Networks (q-bio.MN)</span>; Dynamical Systems (math.DS)
       </div>
      </div>
     </dd>
     <dt>
      <a name="item17">[17]</a>&nbsp; <span class="list-identifier"><a href="/abs/2310.19481" title="Abstract">arXiv:2310.19481</a> (replaced) [<a href="/pdf/2310.19481" title="Download PDF">pdf</a>, <a href="/format/2310.19481" title="Other formats">other</a>]</span>
     </dt>
     <dd>
      <div class="meta">
       <div class="list-title mathjax">
        <span class="descriptor">Title:</span> Spatial information allows inference of the prevalence of direct cell-to-cell viral infection
       </div>
       <div class="list-authors">
        <span class="descriptor">Authors:</span> <a href="/search/q-bio?searchtype=author&amp;query=Williams%2C+T">Thomas Williams</a>, <a href="/search/q-bio?searchtype=author&amp;query=McCaw%2C+J+M">James M. McCaw</a>, <a href="/search/q-bio?searchtype=author&amp;query=Osborne%2C+J">James Osborne</a>
       </div>
       <div class="list-subjects">
        <span class="descriptor">Subjects:</span> <span class="primary-subject">Quantitative Methods (q-bio.QM)</span>; Tissues and Organs (q-bio.TO)
       </div>
      </div>
     </dd>
     <dt>
      <a name="item18">[18]</a>&nbsp; <span class="list-identifier"><a href="/abs/2311.07338" title="Abstract">arXiv:2311.07338</a> (replaced) [<a href="/pdf/2311.07338" title="Download PDF">pdf</a>, <a href="/format/2311.07338" title="Other formats">other</a>]</span>
     </dt>
     <dd>
      <div class="meta">
       <div class="list-title mathjax">
        <span class="descriptor">Title:</span> A mathematical model of the visual MacKay effect
       </div>
       <div class="list-authors">
        <span class="descriptor">Authors:</span> <a href="/search/math?searchtype=author&amp;query=Tamekue%2C+C">Cyprien Tamekue</a>, <a href="/search/math?searchtype=author&amp;query=Prandi%2C+D">Dario Prandi</a>, <a href="/search/math?searchtype=author&amp;query=Chitour%2C+Y">Yacine Chitour</a>
       </div>
       <div class="list-subjects">
        <span class="descriptor">Subjects:</span> <span class="primary-subject">Optimization and Control (math.OC)</span>; Analysis of PDEs (math.AP); Numerical Analysis (math.NA); Neurons and Cognition (q-bio.NC)
       </div>
      </div>
     </dd>
     <dt>
      <a name="item19">[19]</a>&nbsp; <span class="list-identifier"><a href="/abs/2311.12410" title="Abstract">arXiv:2311.12410</a> (replaced) [<a href="/pdf/2311.12410" title="Download PDF">pdf</a>, <a href="/format/2311.12410" title="Other formats">other</a>]</span>
     </dt>
     <dd>
      <div class="meta">
       <div class="list-title mathjax">
        <span class="descriptor">Title:</span> nach0: Multimodal Natural and Chemical Languages Foundation Model
       </div>
       <div class="list-authors">
        <span class="descriptor">Authors:</span> <a href="/search/cs?searchtype=author&amp;query=Livne%2C+M">Micha Livne</a>, <a href="/search/cs?searchtype=author&amp;query=Miftahutdinov%2C+Z">Zulfat Miftahutdinov</a>, <a href="/search/cs?searchtype=author&amp;query=Tutubalina%2C+E">Elena Tutubalina</a>, <a href="/search/cs?searchtype=author&amp;query=Kuznetsov%2C+M">Maksim Kuznetsov</a>, <a href="/search/cs?searchtype=author&amp;query=Polykovskiy%2C+D">Daniil Polykovskiy</a>, <a href="/search/cs?searchtype=author&amp;query=Brundyn%2C+A">Annika Brundyn</a>, <a href="/search/cs?searchtype=author&amp;query=Jhunjhunwala%2C+A">Aastha Jhunjhunwala</a>, <a href="/search/cs?searchtype=author&amp;query=Costa%2C+A">Anthony Costa</a>, <a href="/search/cs?searchtype=author&amp;query=Aliper%2C+A">Alex Aliper</a>, <a href="/search/cs?searchtype=author&amp;query=Aspuru-Guzik%2C+A">Alán Aspuru-Guzik</a>, <a href="/search/cs?searchtype=author&amp;query=Zhavoronkov%2C+A">Alex Zhavoronkov</a>
       </div>
       <div class="list-comments mathjax">
        <span class="descriptor">Comments:</span> Accepted to Chemical Science Journal
       </div>
       <div class="list-subjects">
        <span class="descriptor">Subjects:</span> <span class="primary-subject">Computation and Language (cs.CL)</span>; Artificial Intelligence (cs.AI); Machine Learning (cs.LG); Quantitative Methods (q-bio.QM)
       </div>
      </div>
     </dd>
     <dt>
      <a name="item20">[20]</a>&nbsp; <span class="list-identifier"><a href="/abs/2402.12785" title="Abstract">arXiv:2402.12785</a> (replaced) [<a href="/pdf/2402.12785" title="Download PDF">pdf</a>, <a href="/format/2402.12785" title="Other formats">other</a>]</span>
     </dt>
     <dd>
      <div class="meta">
       <div class="list-title mathjax">
        <span class="descriptor">Title:</span> Stochastic Graph Heat Modelling for Diffusion-based Connectivity Retrieval
       </div>
       <div class="list-authors">
        <span class="descriptor">Authors:</span> <a href="/search/eess?searchtype=author&amp;query=Goerttler%2C+S">Stephan Goerttler</a>, <a href="/search/eess?searchtype=author&amp;query=He%2C+F">Fei He</a>, <a href="/search/eess?searchtype=author&amp;query=Wu%2C+M">Min Wu</a>
       </div>
       <div class="list-comments mathjax">
        <span class="descriptor">Comments:</span> 4 pages, 1 figure, conference paper
       </div>
       <div class="list-subjects">
        <span class="descriptor">Subjects:</span> <span class="primary-subject">Signal Processing (eess.SP)</span>; Neurons and Cognition (q-bio.NC); Methodology (stat.ME)
       </div>
      </div>
     </dd>
     <dt>
      <a name="item21">[21]</a>&nbsp; <span class="list-identifier"><a href="/abs/2402.13523" title="Abstract">arXiv:2402.13523</a> (replaced) [<a href="/pdf/2402.13523" title="Download PDF">pdf</a>, <a href="/format/2402.13523" title="Other formats">other</a>]</span>
     </dt>
     <dd>
      <div class="meta">
       <div class="list-title mathjax">
        <span class="descriptor">Title:</span> Balancing Spectral, Temporal and Spatial Information for EEG-based Alzheimer's Disease Classification
       </div>
       <div class="list-authors">
        <span class="descriptor">Authors:</span> <a href="/search/eess?searchtype=author&amp;query=Goerttler%2C+S">Stephan Goerttler</a>, <a href="/search/eess?searchtype=author&amp;query=He%2C+F">Fei He</a>, <a href="/search/eess?searchtype=author&amp;query=Wu%2C+M">Min Wu</a>
       </div>
       <div class="list-comments mathjax">
        <span class="descriptor">Comments:</span> 4 pages, 3 figures, conference paper
       </div>
       <div class="list-subjects">
        <span class="descriptor">Subjects:</span> <span class="primary-subject">Signal Processing (eess.SP)</span>; Machine Learning (cs.LG); Neurons and Cognition (q-bio.NC)
       </div>
      </div>
     </dd>
     <dt>
      <a name="item22">[22]</a>&nbsp; <span class="list-identifier"><a href="/abs/2404.08423" title="Abstract">arXiv:2404.08423</a> (replaced) [<a href="/pdf/2404.08423" title="Download PDF">pdf</a>, <a href="/format/2404.08423" title="Other formats">other</a>]</span>
     </dt>
     <dd>
      <div class="meta">
       <div class="list-title mathjax">
        <span class="descriptor">Title:</span> SIR-RL: Reinforcement Learning for Optimized Policy Control during Epidemiological Outbreaks in Emerging Market and Developing Economies
       </div>
       <div class="list-authors">
        <span class="descriptor">Authors:</span> <a href="/search/cs?searchtype=author&amp;query=Jain%2C+M">Maeghal Jain</a>, <a href="/search/cs?searchtype=author&amp;query=Uddin%2C+Z">Ziya Uddin</a>, <a href="/search/cs?searchtype=author&amp;query=Ibrahim%2C+W">Wubshet Ibrahim</a>
       </div>
       <div class="list-comments mathjax">
        <span class="descriptor">Comments:</span> 27 pages, 12 figures
       </div>
       <div class="list-subjects">
        <span class="descriptor">Subjects:</span> <span class="primary-subject">Machine Learning (cs.LG)</span>; Physics and Society (physics.soc-ph); Populations and Evolution (q-bio.PE)
       </div>
      </div>
     </dd>
     <dt>
      <a name="item23">[23]</a>&nbsp; <span class="list-identifier"><a href="/abs/2404.11403" title="Abstract">arXiv:2404.11403</a> (replaced) [<a href="/pdf/2404.11403" title="Download PDF">pdf</a>, <a href="/format/2404.11403" title="Other formats">other</a>]</span>
     </dt>
     <dd>
      <div class="meta">
       <div class="list-title mathjax">
        <span class="descriptor">Title:</span> Six decades of the FitzHugh-Nagumo model: A guide through its spatio-temporal dynamics and influence across disciplines
       </div>
       <div class="list-authors">
        <span class="descriptor">Authors:</span> <a href="/search/nlin?searchtype=author&amp;query=Cebri%C3%A1n-Lacasa%2C+D">Daniel Cebrián-Lacasa</a>, <a href="/search/nlin?searchtype=author&amp;query=Parra-Rivas%2C+P">Pedro Parra-Rivas</a>, <a href="/search/nlin?searchtype=author&amp;query=Ruiz-Reyn%C3%A9s%2C+D">Daniel Ruiz-Reynés</a>, <a href="/search/nlin?searchtype=author&amp;query=Gelens%2C+L">Lendert Gelens</a>
       </div>
       <div class="list-comments mathjax">
        <span class="descriptor">Comments:</span> 38 pages, 18 figures
       </div>
       <div class="list-subjects">
        <span class="descriptor">Subjects:</span> <span class="primary-subject">Pattern Formation and Solitons (nlin.PS)</span>; Mathematical Physics (math-ph); Biological Physics (physics.bio-ph); Computational Physics (physics.comp-ph); Other Quantitative Biology (q-bio.OT)
       </div>
      </div>
     </dd>
     <dt>
      <a name="item24">[24]</a>&nbsp; <span class="list-identifier"><a href="/abs/2404.15369" title="Abstract">arXiv:2404.15369</a> (replaced) [<a href="/pdf/2404.15369" title="Download PDF">pdf</a>, <a href="/format/2404.15369" title="Other formats">other</a>]</span>
     </dt>
     <dd>
      <div class="meta">
       <div class="list-title mathjax">
        <span class="descriptor">Title:</span> Can a Machine be Conscious? Towards Universal Criteria for Machine Consciousness
       </div>
       <div class="list-authors">
        <span class="descriptor">Authors:</span> <a href="/search/q-bio?searchtype=author&amp;query=Anwar%2C+N+A">Nur Aizaan Anwar</a>, <a href="/search/q-bio?searchtype=author&amp;query=Badea%2C+C">Cosmin Badea</a>
       </div>
       <div class="list-comments mathjax">
        <span class="descriptor">Comments:</span> This work was supported by the UKRI CDT in AI for Healthcare, <a href="http://ai4health.io">this http URL</a> (Grant No. EP/S023283/1)
       </div>
       <div class="list-subjects">
        <span class="descriptor">Subjects:</span> <span class="primary-subject">Neurons and Cognition (q-bio.NC)</span>; Artificial Intelligence (cs.AI); Computers and Society (cs.CY)
       </div>
      </div>
     </dd>
     <dt>
      <a name="item25">[25]</a>&nbsp; <span class="list-identifier"><a href="/abs/2404.17454" title="Abstract">arXiv:2404.17454</a> (replaced) [<a href="/pdf/2404.17454" title="Download PDF">pdf</a>, <a href="/format/2404.17454" title="Other formats">other</a>]</span>
     </dt>
     <dd>
      <div class="meta">
       <div class="list-title mathjax">
        <span class="descriptor">Title:</span> Domain Adaptive and Fine-grained Anomaly Detection for Single-cell Sequencing Data and Beyond
       </div>
       <div class="list-authors">
        <span class="descriptor">Authors:</span> <a href="/search/cs?searchtype=author&amp;query=Xu%2C+K">Kaichen Xu</a>, <a href="/search/cs?searchtype=author&amp;query=Ding%2C+Y">Yueyang Ding</a>, <a href="/search/cs?searchtype=author&amp;query=Hou%2C+S">Suyang Hou</a>, <a href="/search/cs?searchtype=author&amp;query=Zhan%2C+W">Weiqiang Zhan</a>, <a href="/search/cs?searchtype=author&amp;query=Chen%2C+N">Nisang Chen</a>, <a href="/search/cs?searchtype=author&amp;query=Wang%2C+J">Jun Wang</a>, <a href="/search/cs?searchtype=author&amp;query=Sun%2C+X">Xiaobo Sun</a>
       </div>
       <div class="list-comments mathjax">
        <span class="descriptor">Comments:</span> 17 pages, 2 figures. Accepted by IJCAI 2024
       </div>
       <div class="list-subjects">
        <span class="descriptor">Subjects:</span> <span class="primary-subject">Machine Learning (cs.LG)</span>; Artificial Intelligence (cs.AI); Quantitative Methods (q-bio.QM)
       </div>
      </div>
     </dd>
    </dl>
    <ul>
     <li><a href="/list/q-bio/new?skip=0&amp;show=2000">New submissions</a></li>
     <li><a href="#item9">Cross-lists</a></li>
     <li><a href="#item14">Replacements</a></li>
    </ul><small>[ total of 25 entries: <b>1-25</b> ]</small>
    <br><small>[ showing up to 2000 entries per page: <a href="/list/q-bio/new?skip=0&amp;show=1000">fewer</a> | <font color="#999999">more</font> ]</small>
    <br>
   </div>
   <br><small><a id="mathjax_toggle" href="javascript:setMathjaxCookie()">Disable MathJax</a> (<a href="/help/mathjax">What is MathJax?</a>)</small>
   <script type="text/javascript" language="javascript">mathjaxToggle();</script>
   <hr>
   <p>Links to: <a href="/" accesskey="a">arXiv</a>, <a href="/form/q-bio">form interface</a>, <a href="/find/q-bio">find</a>, <a href="/archive/q-bio">q-bio</a>, <a href="/list/q-bio/recent">recent</a>, <a href="/list/q-bio/2404">2404</a>, <a href="/help/contact">contact</a>, <a href="/help/" accesskey="h"><span class="accesskey">h</span>elp</a>&nbsp; <small>(<a href="/help/accesskeys">Access key</a> information)</small></p>
   <hr>
  </div>
  <footer style="clear: both;">
   <div class="columns is-desktop" role="navigation" aria-label="Secondary" style="margin: -0.75em -0.75em 0.75em -0.75em">
    <!-- Macro-Column 1 -->
    <div class="column" style="padding: 0;">
     <div class="columns">
      <div class="column">
       <ul style="list-style: none; line-height: 2;">
        <li><a href="https://arxiv.org/about">About</a></li>
        <li><a href="https://arxiv.org/help">Help</a></li>
       </ul>
      </div>
      <div class="column">
       <ul style="list-style: none; line-height: 2;">
        <li>
         <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="icon filter-black" role="presentation">
          <title>
           contact arXiv
          </title><desc>
           Click here to contact arXiv
          </desc><path d="M502.3 190.8c3.9-3.1 9.7-.2 9.7 4.7V400c0 26.5-21.5 48-48 48H48c-26.5 0-48-21.5-48-48V195.6c0-5 5.7-7.8 9.7-4.7 22.4 17.4 52.1 39.5 154.1 113.6 21.1 15.4 56.7 47.8 92.2 47.6 35.7.3 72-32.8 92.3-47.6 102-74.1 131.6-96.3 154-113.7zM256 320c23.2.4 56.6-29.2 73.4-41.4 132.7-96.3 142.8-104.7 173.4-128.7 5.8-4.5 9.2-11.5 9.2-18.9v-19c0-26.5-21.5-48-48-48H48C21.5 64 0 85.5 0 112v19c0 7.4 3.4 14.3 9.2 18.9 30.6 23.9 40.7 32.4 173.4 128.7 16.8 12.2 50.2 41.8 73.4 41.4z" />
         </svg><a href="https://arxiv.org/help/contact"> Contact</a></li>
        <li>
         <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="icon filter-black" role="presentation">
          <title>
           subscribe to arXiv mailings
          </title><desc>
           Click here to subscribe
          </desc><path d="M476 3.2L12.5 270.6c-18.1 10.4-15.8 35.6 2.2 43.2L121 358.4l287.3-253.2c5.5-4.9 13.3 2.6 8.6 8.3L176 407v80.5c0 23.6 28.5 32.9 42.5 15.8L282 426l124.6 52.2c14.2 6 30.4-2.9 33-18.2l72-432C515 7.8 493.3-6.8 476 3.2z" />
         </svg><a href="https://arxiv.org/help/subscribe"> Subscribe</a></li>
       </ul>
      </div>
     </div>
    </div><!-- End Macro-Column 1 --> <!-- Macro-Column 2 -->
    <div class="column" style="padding: 0;">
     <div class="columns">
      <div class="column">
       <ul style="list-style: none; line-height: 2;">
        <li><a href="https://arxiv.org/help/license">Copyright</a></li>
        <li><a href="https://arxiv.org/help/policies/privacy_policy">Privacy Policy</a></li>
       </ul>
      </div>
      <div class="column sorry-app-links">
       <ul style="list-style: none; line-height: 2;">
        <li><a href="https://arxiv.org/help/web_accessibility">Web Accessibility Assistance</a></li>
        <li>
         <p class="help"><a class="a11y-main-link" href="https://status.arxiv.org" target="_blank">arXiv Operational Status 
           <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 256 512" class="icon filter-dark_grey" role="presentation">
            <path d="M224.3 273l-136 136c-9.4 9.4-24.6 9.4-33.9 0l-22.6-22.6c-9.4-9.4-9.4-24.6 0-33.9l96.4-96.4-96.4-96.4c-9.4-9.4-9.4-24.6 0-33.9L54.3 103c9.4-9.4 24.6-9.4 33.9 0l136 136c9.5 9.4 9.5 24.6.1 34z" />
           </svg></a><br>
           Get status notifications via <a class="is-link" href="https://subscribe.sorryapp.com/24846f03/email/new" target="_blank">
           <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" class="icon filter-black" role="presentation">
            <path d="M502.3 190.8c3.9-3.1 9.7-.2 9.7 4.7V400c0 26.5-21.5 48-48 48H48c-26.5 0-48-21.5-48-48V195.6c0-5 5.7-7.8 9.7-4.7 22.4 17.4 52.1 39.5 154.1 113.6 21.1 15.4 56.7 47.8 92.2 47.6 35.7.3 72-32.8 92.3-47.6 102-74.1 131.6-96.3 154-113.7zM256 320c23.2.4 56.6-29.2 73.4-41.4 132.7-96.3 142.8-104.7 173.4-128.7 5.8-4.5 9.2-11.5 9.2-18.9v-19c0-26.5-21.5-48-48-48H48C21.5 64 0 85.5 0 112v19c0 7.4 3.4 14.3 9.2 18.9 30.6 23.9 40.7 32.4 173.4 128.7 16.8 12.2 50.2 41.8 73.4 41.4z" />
           </svg>email</a> or <a class="is-link" href="https://subscribe.sorryapp.com/24846f03/slack/new" target="_blank">
           <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512" class="icon filter-black" role="presentation">
            <path d="M94.12 315.1c0 25.9-21.16 47.06-47.06 47.06S0 341 0 315.1c0-25.9 21.16-47.06 47.06-47.06h47.06v47.06zm23.72 0c0-25.9 21.16-47.06 47.06-47.06s47.06 21.16 47.06 47.06v117.84c0 25.9-21.16 47.06-47.06 47.06s-47.06-21.16-47.06-47.06V315.1zm47.06-188.98c-25.9 0-47.06-21.16-47.06-47.06S139 32 164.9 32s47.06 21.16 47.06 47.06v47.06H164.9zm0 23.72c25.9 0 47.06 21.16 47.06 47.06s-21.16 47.06-47.06 47.06H47.06C21.16 243.96 0 222.8 0 196.9s21.16-47.06 47.06-47.06H164.9zm188.98 47.06c0-25.9 21.16-47.06 47.06-47.06 25.9 0 47.06 21.16 47.06 47.06s-21.16 47.06-47.06 47.06h-47.06V196.9zm-23.72 0c0 25.9-21.16 47.06-47.06 47.06-25.9 0-47.06-21.16-47.06-47.06V79.06c0-25.9 21.16-47.06 47.06-47.06 25.9 0 47.06 21.16 47.06 47.06V196.9zM283.1 385.88c25.9 0 47.06 21.16 47.06 47.06 0 25.9-21.16 47.06-47.06 47.06-25.9 0-47.06-21.16-47.06-47.06v-47.06h47.06zm0-23.72c-25.9 0-47.06-21.16-47.06-47.06 0-25.9 21.16-47.06 47.06-47.06h117.84c25.9 0 47.06 21.16 47.06 47.06 0 25.9-21.16 47.06-47.06 47.06H283.1z" />
           </svg>slack</a></p></li>
       </ul>
      </div>
     </div>
    </div><!-- end MetaColumn 2 --> <!-- End Macro-Column 2 -->
   </div>
  </footer>
 </body>
</html>